soccer/gameplay/plays/restarts/placement.py

```diff
@@ -59,8 +59,8 @@ def on_enter_placing(self):
                                  required=True,
                                  priority=90)
 
-        #line_up = tactics.line_up.LineUp(self.create_lineup())
-        #self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
+        line_up = tactics.line_up.LineUp(self.create_lineup())
+        self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
```

Comment:
From how I am interpreting the rules, it seems like we don't get any free time to reposition after ball placement. Is this your guys' interpretation as well?